src/models/regulation_device.jl

```diff
@@ -5,7 +5,7 @@ mutable struct RegulationDevice{T <: StaticInjection} <: Device
     reserve_limit::Float64
     inertia::Float64
 
-    function RegulationDevice(
+    function RegulationDevice{T}(
```

Comment:
We should maintain a method that isn't parametrized to create regulation devices easily